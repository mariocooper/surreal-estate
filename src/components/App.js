import NavBar from './NavBar';
import '../styles/app.css';

function App() {
  return (
    <div className="app">
      <header className="app-header">
        Surreal Estate
      </header>
      <NavBar />
    </div>
  );
}

export default App;
