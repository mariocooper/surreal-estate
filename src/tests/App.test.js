import { BrowserRouter } from 'react-router-dom';
import { render } from '@testing-library/react';

import App from '../components/App';

describe('App', () => {
  it('renders app correctly', () => {
    const { asFragment } = render(
    <BrowserRouter>
      <App />
    </BrowserRouter>
    );
    expect(asFragment()).toMatchSnapshot();
  });
});
