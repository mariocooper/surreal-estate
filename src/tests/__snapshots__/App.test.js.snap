// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App renders app correctly 1`] = `
<DocumentFragment>
  <div
    class="app"
  >
    <header
      class="app-header"
    >
      Surreal Estate
    </header>
    <div
      class="navbar"
    >
      <img
        alt="surrealEstateLogo"
        class="navbar-image"
        src="https://mcrcodes.s3.eu-west-2.amazonaws.com/course/surreal-estate/logo.png"
      />
      <ul>
        <a
          class="navbar-links-item"
          href="/"
        >
          View Properties
        </a>
        <a
          class="navbar-links-item"
          href="/"
        >
          Add a Property
        </a>
      </ul>
    </div>
  </div>
</DocumentFragment>
`;
